src/scss/components/layout/_index.scss

```diff
@@ -1,9 +1,8 @@
-@import (reference) 'vars';
-@import (reference) 'components/box/index';
+@import 'vars';
 
 // NOTE: be aware of changes needed in the base/typography.less
-@header-height: 3.5rem; // 56px
-@footer-height: 2.5rem; // 40px
+$header-height: 3.5rem; // 56px
+$footer-height: 2.5rem; // 40px
 
 /* ======================================= *\
```

Comment:
Please update this layout comment block [Line 7-Line 38] to use three slash comments`///` per line, so that it can used in our documentation.

Reply:
Please check the changes in latest commit